doc/source/cookbook.rst

```diff
@@ -1236,7 +1236,7 @@ the following Python code will read the binary file ``'binary.dat'`` into a
 pandas ``DataFrame``, where each element of the struct corresponds to a column
 in the frame:
 
-.. code-block:: python
+.. ipython:: python
```

Comment:
Is there a reason you changed this? As this doesn't run (there is no file `binary.dat`)

Reply:
I reverted it in https://github.com/pandas-dev/pandas/pull/24552 (along with some other doc fixes), but we certainly can also try to actually fix the example (meaning: make it runnable).

Reply:
I found it inconsistent that everything had the ipython formatting and not this block (when rendered), and I thought `code-block` had to have the `>>>`, and that not being an `ipython` block was an error.

But it's ok to have them reverted. At least for now.

Reply:
It was indeed a bit strange to just include code without output (cookbook is not very well maintained anyhow).
We could in principle also include IPython like formatting in a code-block, if that makes it a bit more consistent.